app/controllers/GerkinController.scala

```diff
@@ -0,0 +1,127 @@
+package controllers
+
+import io.swagger.annotations._
+import javax.inject.Inject
+import models._
+import play.api.Logging
+import play.api.libs.json.Json
+import play.api.mvc._
+import repository._
+import services.MenuService
+
+case class BranchGerkinDTO(id: Long, name: String, isStable: Boolean, features: Seq[Feature])
```

Comment:
Move the DTOs in a subpackage controllers.dto, and rename them to Documentation instead of Gerkin